### PR TITLE
Add `bitrise.yml` to repository

### PR DESCRIPTION
**Pull request by gersonnoboa**
In order to have changes to the `bitrise.yml` go through source
control, the file is added to the repository. No changes have been
done to this file, as it has just been downloaded directly from
Bitrise.

MOB-1510

### CODE REVIEW

**Review by AndriiHorishniiMOC — APPROVED** [review]
Is it OK to have this file in the open repository?

**Comment by gersonnoboa** [comment]
Yes, I investigated about it and the bitrise.yml itself doesn't show sensitive information, because secrets are stored in Bitrise itself and just referenced in the yml [like this](https://github.com/salemove/android-sdk-widgets/pull/348/files#diff-61d1ee831cb1df242161a7d23171914a1d5d628c7659b52db957d80a9394bd68R21).

File: bitrise.yml [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+    - deploy-to-bitrise-io@1:
+        inputs:
+        - notify_email_list: $BUILD_EMAILS
+    - cache-push@2: {}
+    envs:
+    - opts:
+        is_expand: false
+      INTEGRATOR_VARIANT: master
+  publish_to_nexus:
+    description: Task builts a SDK and upload it to Nexus.
```
Comment (andrews-moc) [inline bitrise.yml:139]: I  think it existed before, but if it's not difficult for you, maybe it's better to change this description to "Task builds an SDK and uploads it to Nexus."

**Review by andrews-moc — APPROVED** [review]

File: bitrise.yml (hunk 1 above)
Comment (andrews-moc) [reply bitrise.yml:139]: Shall I do it on my own?

**Review by DavDo — APPROVED** [review]

File: bitrise.yml (hunk 1 above)
Comment (gersonnoboa) [reply bitrise.yml:139]: Damn I didn't see this, I will open a quick PR to amend this.

**Pull request merged**